html5validator/validator.py

```diff
@@ -150,8 +152,26 @@ def validate(self, files):
             regex = re.compile(i)
             e = [l for l in e if not regex.search(l)]
 
-        if stderr:
-            LOGGER.error(stderr)
+        if e:
+            LOGGER.debug(e)
+            if self.format == "text":
+                LOGGER.error(e)
+                return len(e)
+            elif self.format == "gnu":
+                LOGGER.error(e)
+                return len(e)
+            elif self.format == "json" and e[0] != u'{"messages":[]}':
```

Comment:
Can this be absorbed into an ignore pattern?

Reply:
The JSON message has been successfully added into the ignore pattern